lib/tooltip.js

```diff
@@ -152,7 +152,7 @@ class Tooltip {
     const currentStep = this.currentStepNum();
     const totalSteps = this.tutorial.steps.length;
     this.cta = this.config.cta || (currentStep != totalSteps ? 'Next' : 'Done');
-    this.subtext = this.config.subtext ||
+    this.subtext = typeof this.config.subtext === 'function' ? this.config.subtext :
```

Comment:
What do you think about doing this check earlier in the constructor and throwing an error if it's the wrong type?